src/datepicker/test/datepicker.spec.js

```diff
@@ -1346,13 +1351,17 @@ describe('datepicker', function() {
 
         expect(getTitle()).toBe('10');
         expect(getOptions()).toEqual([
-          ['Jan', 'Feb', 'Mar'],
-          ['Apr', 'May', 'Jun'],
-          ['Jul', 'Aug', 'Sep'],
-          ['Oct', 'Nov', 'Dec']
+          ['Jan', 'Feb', 'Mar', 'Apr'],
+          ['May', 'Jun', 'Jul', 'Aug'],
+          ['Sep', 'Oct', 'Nov', 'Dec']
         ]);
       });
 
+      it('changes colspan of yearbutton in month selector to have colspan', function() {
```

Comment:
Test name should be descriptive on what is happening at a high level, not specifics about implementation